### PR TITLE
Add stable release info, remove old submodule URLs

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,32 +9,16 @@ May install an example puppet master setup using passenger as well, including th
...
+published via the normal Foreman repositories.
 
-  * http://github.com/theforeman/puppet-apache/tarball/master
-  * http://github.com/theforeman/puppet-foreman/tarball/master
-  * http://github.com/theforeman/puppet-foreman_proxy/tarball/master
-  * http://github.com/theforeman/puppet-passenger/tarball/master
-  * http://github.com/theforeman/puppet-puppet/tarball/master
-  * http://github.com/theforeman/puppet-tftp/tarball/master
-  * http://github.com/theforeman/puppet-xinetd/tarball/master
+See the manual and quickstart guide available at [theforeman.org](http://theforeman.org/).
```
Comment (skottler) [inline README.md:17]: What about just linking directly to `http://theforeman.org/manuals/1.1/index.html#3.2ForemanInstaller`?

Comment (domcleal) [inline README.md:17]: The lack of static links put me off :)  It won't be long before we're on 1.2!

Comment (skottler) [inline README.md:17]: True, I just think there will be a lot of people who abandon the page if they get directed to the index page of the site. That said, the versioned pages make maintaining permalinks tough so this is :+1: as it is from me.

**Comment by GregSutcliffe** [comment]
:+1: although you've reminded me I need to review the readme.debian file, as it's woefully outdated.

**Pull request merged**